packages/trpc/server/routers/viewer.tsx

```diff
@@ -30,6 +33,8 @@ import slugify from "@calcom/lib/slugify";
 import prisma, { baseEventTypeSelect, bookingMinimalSelect } from "@calcom/prisma";
 import { resizeBase64Image } from "@calcom/web/server/lib/resizeBase64Image";
 
+import { ErrorCode, verifyPassword } from "@lib/auth";
```

Comment:
We cannot use `@lib` outside `apps/web`

Reply:
Should I move the ones I need to re-use in trpc to new packages/auth or just packages/lib/auth.ts?